File: models/feedback.py
from companies import Company
from django.db import models

class Feedback(models.Model):
	customer_firstname = models.CharField(max_length=30)
	customer_lastname = models.CharField(max_length=30)
	customer_email = models.EmailField()
	comments = models.TextField()
	approved = models.BooleanField(default=True)
	show_name_public = models.BooleanField(default=False)
	#one to many relation to the company
	company = models.ForeignKey(Company)

	class Meta:
		app_label = "carpathian"
		db_table = "carpathian_feedback"
		verbose_name_plural = "feedback"

	def __unicode__(self):
		return self.customer_firstname+" "+self.customer_lastname+" on "+self.company.name

File: static/js/models/Feedback.js
CP.Feedback = Backbone.Model.extend({

}, 
{
	create: function(user, comments){
		var feedback = new CP.Feedback({ user:user, comments:comments });
		CP.feedbackList.add(feedback);

		return feedback;
	},

	create: function(user, comments, approved, id){
		var feedback = new CP.Feedback({ user:user, comments:comments, approved:approved, id:id });
		CP.feedbackList.add(feedback);

		return feedback;
	}
});

CP.FeedbackList = Backbone.Collection.extend({
	model: CP.Feedback,
});

CP.feedbackList = new CP.FeedbackList();